docs/archive_data_migration_to_berkeley.md

```diff
@@ -0,0 +1,470 @@
+2. Next, run replayer app in migration mode. 
+
+The applications are able to work incrementally so that part of the Mainnet database can be migrated and, as new blocks are added to the Mainnet, the new data in the database can be migrated.
+
+To obtain that incrementality, the berkeley-migration application can look at the migrated database and determine the most recent migrated block. It can continue migrating starting at the next block in the Mainnet data. The replayer app in migration mode can use the checkpoint mechanism already in place for the replayer. A checkpoint file indicates the global slot since genesis for starting the replay and the ledger to use for that replay. The application writes new checkpoint files as it proceeds.
+
+To take advantage of the incrementality, you can run a cron job that migrates a day's worth of data at a time (or some other interval). With the cron job in place, at the time of the actual hard fork, only a small amount of data will need to be migrated.
+
+## Requirements
+
+To successfully migrate the Mainnet database into the Berkeley version of the Mina network, the foundational requirements are:
+
+### Database server
+
+- Database: PostgreSQL database in version 15.0 or later.
+
+### Migration machine
+
+- (Optional) Docker in version 23.0 or later
+- If Docker is used, then any of the supported OS by Mina (Bullseye, Focal, or Buster)
+at least 32 GB of RAM
+- gsutil application from Google Cloud Suite in version 5 or later
+
+## Prerequisites
+
+### (Optional) Download and import Mainnet dump
+
+If you don't have an existing database with Mainnet archive data, you can always download it from our Google Cloud bucket. 
+
+1. Download the Mainnet archive data using cURL or gsutils:
+
+   - To use cURL to download the Tar archive:
+
+   ```sh
+   curl https://storage.googleapis.com/mina-archive-dumps/mainnet-archive-dump-{date}_0000.sql.tar.gz
+   ```
+
+   You can filter the dumps by date. Replace `{date}` using the required yyyy-dd-mm format. For example, for January 15, 2024: `2024-01-15`
+
+   > :warning: In some cases, the 0000 suffix in the date might be different (0001).
+
+
+   - To use gsutils, change the date in this example:
+   
+      ```sh
+      gsutil cp gs://mina-archive-dumps/mainnet-archive-dump-2024-01-15* .
+      ```
+
+2. Extract the tar package.
+
+3. Import the Mainnet archive dump into the Berkeley database.
+
+   Run this command at the database server:
+
+   `psql -U {user} -f mainnet-archive-dump-{date}_0000.sql`
+
+   An *archive_balances_migrated* schema is created with the Mainnet archive.
+
+### Validate the Mainnet database
+
+The correct Mainnet database state is crucial for a successful migration. 
+
+Missing blocks or invalid ledger hashes are the most frequent issues when dealing with the Mainnet archive. Although this step is optional, it is strongly recommended to verify the archive condition before you start the migration process.
+
+### Known issues
```

Comment:
these known issues are for which phase? We have another Known issues on line 403 

Do we want to isolate known issues for each section (part of the migration?) or put all together? 

https://github.com/MinaProtocol/mina/pull/14933/files#diff-677a92f8728b33ec9935394888590b6fe92c6157d3db38b1fda79e798ecc68e0R403